ansible_base/jwt_consumer/common/auth.py

```diff
@@ -200,11 +212,16 @@ def process_rbac_permissions(self):
             logger.error("Unable to process rbac permissions because user or token is not defined, please call authenticate first")
             return
 
+        role_diff = RoleUserAssignment.objects.filter(
+            user=self.user, role_definition__name__in=[role.name for role in permission_registry()._managed_roles.values()]
```

Comment:
I wish I had a better answer, but I think we're just going to have to hard-code these. There is a distinction here between "external" (from the JWT) and "managed". Even if everything made use of the "managed" role registry in the same way right now (which they don't), what we care about is "external" roles, where the JWT manages those roles. There's no standard registry of those at all.

It's probably best to put this into a setting, but that setting, I think is just going to have to list ["Organization Member", ...